### ISSUE TITLE
Closing horizontal split pane causes rendering issues

### ISSUE THREAD

**Issue by abronan** [zeit/hyper#1676]
- **OS version and name**: macOS Sierra 10.12.3
- **Hyper.app version**: 1.3.1
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: [.hyper.js](https://gist.github.com/abronan/2fbb0563ecab9f978b3224c916a979c6.js)
- **Relevant information from devtools**: Console on the gif below.
- **The issue is reproducible in vanilla Hyper.app**: Vanilla where only modifications are a custom font type (Hasklig) and size (12) (although the app updated a few times already, I bet this matters)

## Issue

A gif is better than a long description 😅 

![kapture 2017-03-17 at 21 21 13](https://cloud.githubusercontent.com/assets/1655859/24044789/b5360546-0b57-11e7-8eb7-17139015ec49.gif)

To sum up:

- Open a terminal
- Use a command like `git diff` (which paginates)
- Open an horizontal split pane
- Close the split pane
- Use `git diff` (or else) again

Rendering issues appears right after closing the horizontal split pane.

**Comment by remy** [comment]
I'm seeing this too, specifically what was shown in issue #1677.

**Comment by psimyn** [comment]
dupe of #1628 ?

**Comment by zsol** [comment]
No, this is separate. Although the root cause might be the same

**Comment by mbhall88** [comment]
I am seeing a similar issue, but with **vertical split**. After closing the vertical split the window still only displays information up to where the split boundary was before it was closed. The screenshot should illustrate this problem.

My version is Hyper 1.4.8 (stable)

![screenshot 2018-01-25 09 22 45](https://user-images.githubusercontent.com/20403931/35380611-b43b8412-01b1-11e8-8b50-80108fdd2ece.png)

**Comment by Stanzilla** [comment]
Hey there, please try again with the new Hyper v2 release and open a new issue if the problem stills exists. Thank you!
